recipes/newrelic.rb

```diff
@@ -6,7 +6,10 @@
 # Copyright 2014, Rackspace
 #
 
-unless node['newrelic']['license'].nil?
+include_recipe 'chef-sugar'
+newrelic_licence = node.deep_fetch('newrelic', 'license')
```

Comment:
This attribute is not defined by default as we only need [newrelic_meetme_plugin][license]. Before newrelic cookbook was included by platformstack